ui/components/ReconciliationGraph.tsx

```diff
@@ -117,17 +117,21 @@ export default styled(ReconciliationGraph)`
 
   .node {
     position: relative;
-    font-size: 14px;
+    font-size: 16px;
```

Comment:
Why did you bump up the font size? Is that why more text cannot be displayed in the larger node size?

Reply:
I think that the issue has more to do with the zoom scaling than anything else - if I leave the font size the same, as soon as you zoom out at all you can't read it. It's a difference of a couple characters unfortunately...I know we're talking about how we're going to change up the graph, so I'm hoping we can address this then.